docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -198,6 +198,50 @@ It quietly succeeds. Now `\d children` shows that the foreign key constraint `ch
 
 Add the specified constraint to the table.
 
+#### [*alter_column_type*]
+
+Change the type of an existing column. If data on disk is required to change, a full table rewrite is needed.
+
+The new column type will be applied automatically to indexes and simple table constraints by reanalyzing the initially provided expression.
```

Comment:
nit: by reanalyzing the initially provided expression.
reanalyzing sounds vague (and a bit confusing because we have `ANALYZE`) -- do we intend to say "casting" or "re-parsing" here?